_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+with additional data extracted from variable substitutions,
+some provided by the platform,
+some within the `amp-analytics` configuration itself:
+
+{% highlight html linenos %}
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview":"https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}&clientId=${clientId(cid-scope)}",
+  },
+  "vars": {
+    "account": "ABC123",
+  },
+  "triggers": {
+    "some-event": {
+      "on": "visible",
+      "request": "pageview",
+      "vars": {
+        "title": "My homepage",
+        "clientId": "user12345"
+      }
+    }
+  }  
+}
+</script>
+</amp-analytics>
+{% endhighlight %}
+
+In the above example,
+the variables, `account`, `title`, and `clientID` are defined
+in the `amp-analytics` configuration.
+The `canonicalUrl` variable isn’t defined in the configuration,
+so it's value gets substituted by the platform.
+
+**Important:** Variable substitution is flexible;
+you can have the same variables defined in different locations,
+and the AMP runtime will parse the values in this order of precedence
+(see [Variable substitution ordering](/docs/guides/analytics/deep_dive_analytics.html#variable-substitution-ordering)).
+
+## User identification
+
+Websites use cookies to track a user’s browsing history;
+cookies tell the browser that a user visited a web page.
+In AMP,
+pages can be served from either a publisher's website or the AMP Cache.
+The publisher’s website and the AMP Cache have different domains.
+For security reasons,
+browsers can (and often will) limit access to another domain’s cookies
+(see also
+[Tracking users across origins](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/cross-origin-tracking.md)).
+
+By default,
+AMP automatically creates a client ID to identify
+that a user has accessed an AMP page,
+regardless if that page is accessed from the publisher’s website
+or the AMP Cache.
+The AMP-generated client ID has a value of `"amp-"`
+followed by a random `base64` encoded string and remains the same
+for the same user if they visit again within one year.
+
+The client ID is always created by AMP
+anytime the user accesses an AMP page directly from the AMP Cache.
+But in the case when the AMP page is accessed directly
+from the publisher’s side,
+the publisher can override the client ID value
+via a cookie read from the first-party context.
+
+To override the AMP-generated client ID,
+use the special string `CLIENT_ID` to add a per document-source-origin
+(the origin of the website where you publish your AMP doc).
+You must include the `cid-scope`,
+the name of the fallback cookie when the document isn’t served
+by the AMP Cache.
+
+For example:
+
+{% highlight html linenos %}
+<amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(cid-scope-cookie-fallback-name)"></amp-pixel>
+{% endhighlight %}
+
+Learn more about client ID substitution,
```

Comment:
I think this part is unclear. When we talk about the user notification, we should talk about the use case, which is: there may be certain circumstances in which the publisher cannot be granted a client ID because they require the user to accept a privacy notice. In this case, you can associate a user notification, via its ID, to a client ID.